### ISSUE TITLE
Defect: Quiz Component Not Accessible Using Keyboard Commands

### ISSUE THREAD

**Issue by dm9965** [all-rit/ALL#224]
# Defect
When testing labs 6 and 7, discovered that when testing for keyboard accessibility, the multiple choice answers are not accessible for the user to tab through. Make it so the user can now tab from the top of the page, through the quiz, down through the bottom of the page using only the keyboard.

# Acceptance Criteria
- [ ] The quiz component is now keyboard accessible throughout each lab
- [ ] Nothing else is affected by the bug fix
